siac/main.go

```diff
@@ -238,10 +238,14 @@ func main() {
 	renterFilesListCmd.Flags().BoolVarP(&renterListVerbose, "verbose", "v", false, "Show additional file info such as redundancy")
 
 	root.AddCommand(gatewayCmd)
-	gatewayCmd.AddCommand(gatewayAddCmd, gatewayRemoveCmd, gatewayAddressCmd, gatewayListCmd)
+	gatewayCmd.AddCommand(gatewayConnectCmd, gatewayDisconnectCmd, gatewayAddressCmd, gatewayListCmd)
 
 	root.AddCommand(consensusCmd)
 
+	// COMPATv0.6.0
+	gatewayCmd.AddCommand(gatewayDeprecatedAddCmd)
+	gatewayCmd.AddCommand(gatewayDeprecatedRemoveCmd)
```

Comment:
It's fine to do it this way because people are expecting a moderate to substantial change with this release.